clients/cooking/src/theme.ts

```diff
@@ -0,0 +1,7 @@
+import { createMuiTheme } from '@material-ui/core';
```

Comment:
Can you import the theme from `@jeeves/common` instead of creating it in this client?

Reply:
Never mind. You're doing that. We can remove this file.